cmd/check.go

```diff
@@ -47,7 +47,7 @@ Flags:
 		var packageManifestList pkgserverv1.PackageManifestList
 		err = psc.ListPackageManifests(context.TODO(), &packageManifestList, checkflags.FilterPackages)
 		if err != nil {
-			return types.Error{Msg: "Unable to list PackageManifests."}
+			return types.Error{Msg: "Unable to list PackageManifests.\n" + err.Error()}
```

Comment:
I imagine when we refactor the filter logic, it will live somewhere around here as its own err check in this prerun function?